schedule/yast/lvm_multipath_x86_64.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name:           lvm_multipath
+description:    >
+  Textmode installation test for lvm partitioning with no spearate home, on multipath with lvm validation.
+vars:
+  DESKTOP: textmode
+  MULTIPATH: 1
+  LVM: 1
+  SEPARATE_HOME: 0
+  HDDMODEL: scsi-hd
+  YUI_REST_API: 1
+schedule:
+  extension_module_selection:
+    - installation/module_registration/skip_module_registration
+  add_on_product:
+    - installation/multipath
+    - installation/add_on_product/skip_install_addons
+  guided_partitioning:
+    - installation/partitioning/select_guided_setup
+    - installation/partitioning/guided_setup/enable_lvm
+    - installation/partitioning/guided_setup/do_not_propose_separate_home
+  default_systemd_target:
+    - installation/installation_settings/validate_default_target
+  system_validation:
+    - console/validate_multipath
+    - console/validate_lvm
+    - console/hostname
```

Comment:
these  3 test modules, hostname...force_schedule_tasks, should be under system_preparation, it was wrong in the original code, could you please change it?